ensembl/production/dbspy/config.py

```diff
@@ -15,7 +15,7 @@
 
 OPENAPI = {
     "title": "DBSpy",
-    "version": "1.0.0",
+    "version": "1.0",
```

Comment:
Is this the OpenAPI version or dbspy? if this is the latter, it should match the VERSION number, right?

Reply:
This is the API version